### DISCUSSION TITLE
AI role

### DISCUSSION THREAD

**Discussion by LanceLake** [DeltaV-Station/Delta-v#1871] [General]
Will we eventually get the AI character? I noticed the official Wizards Den has it, but we don't. Is this something that we are going to eventually have or has the admins said no already?

**Comment by TadJohnson00** [comment]
We'll pull AI when it reaches a more stable level of development, and when we have the infrastructure in place on our maps; ideally we don't have to pull major fixes and content updates several times a week for such a significant role, nor have to flounder with maps that aren't equipped to handle AI.

**Comment by Sifb26** [comment]
We will have it once most of our maps are properly fitted for an AI core. We also want to wait for them to clean it up a bit. But we are fully planning on implementing it at some point.

**Comment by deltanedas** [comment]
i think around when JUDGE is merged itll be a good point to fully roll out AI to every station

**Comment by LanceLake** [comment]
Coolcool. I've been grinding on Wizard for that role, and I've not been having a fun time (I enjoy Delta much more). So since that's my favorite role, I'm glad to hear that it will be implemented eventually. :)
